Cli/TapInitializer.cs

```diff
@@ -26,63 +25,37 @@ public void EventsLogged(IEnumerable<Event> events)
             public void Flush(){
 
             }
-            public static InitTraceListener Instance = new InitTraceListener();  
+            public static readonly InitTraceListener Instance = new InitTraceListener();  
+        }
+
+        static IEnumerable<string> OpenTapLocations
+        {
+            get
+            {
+                var loc = Path.GetDirectoryName(Assembly.GetEntryAssembly().Location);
+                yield return Path.Combine(loc, "OpenTap.dll");
+            }
```

Comment:
Isn't there always only one item in this enumerable?

Reply:
I've simplified it.